codyze-core/src/main/kotlin/de/fraunhofer/aisec/codyze/Project.kt

```diff
@@ -96,6 +97,16 @@ class AnalysisProject(
     var securityGoalsFolder: Path? = projectDir?.resolve("security-goals"),
     /** The folder where the components are located. */
     var componentsFolder: Path? = projectDir?.resolve("components"),
+    /**
+     * The folder where the additional Sources are located. The subfolders are used to create
```

Comment:
typo: Sources -> sources
should we talk about "libraries" instead of additionalSources in the "format"?
maybe just also name the variable "librariesPath"?

Reply:
Jepp we can do so